app/web_conn.go

```diff
@@ -35,6 +35,7 @@ type WebConn struct {
 	Locale                    string
 	AllChannelMembers         map[string]string
 	LastAllChannelMembersTime int64
+	Sequence                  uint64
```

Comment:
No reason, it's always going to be positive so I went with uint64, I'm 0/5 though